data/com.github.geigi.cozy.appdata.xml

```diff
@@ -224,11 +229,22 @@
       </description>
     </release>
   </releases>
-
-  <content_rating type="oars-1.1" />
+  <provides>
+    <binary>com.github.geigi.cozy</binary>
+  </provides>
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+  <recommends>
+    <display_length compare="gt">360</display_length>
+  </recommends>
+  <branding>
+    <color type="primary" scheme_preference="light">#ffa348</color>
+    <color type="primary" scheme_preference="dark">#ffa348</color>
```

Comment:
I'm not sure how this would actually look like, but these are the colors used on Cozy's icon.

Reply:
Let's have a look how it will look on Flathub & co :)